### DISCUSSION TITLE
notes/trivikram-kamat/

### DISCUSSION THREAD

**Discussion by giscus** [dendronhq/cascadia-js-2022#42] [Announcements]
# notes/trivikram-kamat/

Notes about the cascadiajs 2022 conference

https://cascadiajs.dendron.so/notes/trivikram-kamat/

**Comment by vim-usds** [comment]
Slides here - https://slides.com/trivikram/cascadiajs-publishing/

**Comment by vim-usds** [comment]
Slides here - https://noti.st/expede/iMGOvv/the-dark-forest-private-crdts-as-global-database
